protocols/plain/README.md

```diff
@@ -0,0 +1,22 @@
+# plain
+
+This directory contain the WASM file for the [plain example from refraction-networking/watm](https://github.com/refraction-networking/watm/tree/master/tinygo/v1/examples/plain). This is only intended for development purposes and shouldn't be used in production.
+
+## Dependencies
+
+- [TinyGo](https://tinygo.org/getting-started/install/)
```

Comment:
This is not a dependency of the WASM, right?  This should go under "Steps for generating the WASM file".